stylesheets/print.styl

```diff
@@ -0,0 +1,139 @@
+body 
+	width 100% !important
```

Comment:
Indents should be 2 spaces. Looks like there are some inconsistent ones below, too.